README.md

```diff
@@ -1,8 +1,12 @@
-[![Go Report Card](https://goreportcard.com/badge/github.com/istio/istio)](https://goreportcard.com/report/github.com/istio/istio)
-[![GoDoc](https://godoc.org/istio.io/istio?status.svg)](https://godoc.org/istio.io/istio)
+<a href="https://istio.io/">
+    <img src="logo/istio-bluelogo-whitebackground-framed.svg" alt="istio logo" title="istio" align="right" width="200" height="200" />
```

Comment:
Also, the `alt` and `title` should be `Istio logo`.

Reply:
okay I'll change it

Reply:
Use the full URL.

```
<img src="https://github.com/istio/istio/logo/istio-bluelogo-whitebackground-framed.svg" alt="Istio logo" title="Istio" width="100" />
```

Reply:
Okay got it :)